tests/samples/test_target_sqlite.py

```diff
@@ -138,6 +140,7 @@ def test_sync_sqlite_to_sqlite(
     new_lines = new_stdout.readlines()
     assert len(orig_lines) > 0, "Orig tap output should not be empty."
     assert len(new_lines) > 0, "(Re-)tapped target output should not be empty."
+    assert orig_lines[0] == new_lines[0]
```

Comment:
This test was incorrect before because the state was not included in the provided state to the second call.
```

  - {"type":"STATE","value":{}}
  + {"type":"STATE","value":{"bookmarks":{"main-t0":{"replication_key":"c1","replication_key_value":55}}}}
```